modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,98 @@
+= Portal
+
+`Unit ability`
+
+Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
+This can be an effective method to deploy reserve forces at the optimum time and place.
+
+Well-known examples include Necron portals and Eldar Webway portals.
+
+---
+
+[NOTE]
+====
+This ability lets you keep detachments outside of the play area at the start of the game, similar to Reserves.
+You can then deploy them later in the game.
+When they arrive, you disembark them into the play area through a unit -- or objective -- that has this ability.
+====
+
+== Data sheet variations of {portal}
+
+Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, in brackets.
+For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.
+
+Mostly, this indicates what other friendly units can actually use the portal -- that is, which kinds of unit you can embark/disembark through that portal during the game.
+
+* `_(INF)_` allows Infantry units to use the {portal}.
+* `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
+* `_(Walker)_` allows Vehicle units that have the {walker} ability to use the {portal}, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that use it:
+
+* `_(Auto)_` means that detachments in Portal Reserves do *not* follow the standard rules for Reserves.
+Instead, from turn two onwards, you must *must* disembark them from a {portal} unit/objective in the Movement phase unless there are not enough portals available, or none of the available portals can disembark all of the units in the detachment.
+* `_(Roll)_` means that detachments in Portal Reserves follow the standard rules for Reserves.
+When a detachment arrives from Portal Reserves, you *must* disembark it from a {portal} unit/objective, if possible.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* When you wish to move the units in a detachment out of a portal, use the normal disembarkation rules as at xref:main-rules:transportation.adoc[].
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+* When you wish to embark or disembark a detachment through a portal, you may choose only Advance or Assault orders.
```

Comment:
Do we need to state something to the effect that Portal Reserves are already embarked, so there is no need to deduct from their movement for embarkation?

Reply:
If you think it's needed. I didn't mention it because when the detachment arrives on the table it's only disembarking.

Reply:
Okay. I think it _might_ be needed, depending on how the overall phrasing works out in comparison to texts that relates to the standard Reserves and transportation rules.

Perhaps I'm over-thinking it, but there were one or two subtle points that made me wonder if some players **might** suppose that the first move from off-table implicitly involves an embark action in a way that regular transportation doesn't -- to enter an imaginary off-table portal before the explicit disembark action to exit the on-table portal, and thus **two** 5 cm deductions to the first move.

By the time I've rephrased and restructured bits, that possibility might diminish anyway; but if it still seems iffy then I'll add something explicit to clarify.